pxe_files/firstboot_scripts/desktop-bootstrap-user.sh

```diff
@@ -205,12 +205,12 @@ fi
   # Create User
   echo 'Configure user / group account'
   
-  grep -q $username /etc/group || {
-    groupadd -g 999 $username
+  grep -q ^admin /etc/group || {
```

Comment:
This regex is too greedy (i.e. matches `administrator`). Instead of regex consider `getent group admin`